app/models/point.rb

```diff
@@ -25,6 +23,9 @@ def self.search_points_by_topic(query)
   Point.joins(:topic).where("topics.title ILIKE ?", "%#{query}%")
 end
 
+def approved_with_case(points)
```

Comment:
Given that this does not do anything with its Point instance (only with points given to it as arguments), I would think this doesn't really belong here... It's not used anywhere, is it? If so, maybe we could just remove it?

Reply:
removed